README.md

```diff
@@ -493,6 +493,12 @@ name,iterations,real_time,cpu_time,bytes_per_second,items_per_second,label
 "BM_SetInsert/1024/10",106365,17238.4,8421.53,4.74973e+06,1.18743e+06,
 ```
 
+## Output Files
+The library supports writing the output of the benchmark to a file specified
+by `--benchmark_output=<filename>`. The format of the output can be specified
+using `--benchmark_output_format={json|console|csv}`. Specifying
+`--benchmark_output` does not suppress the console output.
```

Comment:
how often do you think someone would want to push the output to tooling and have it human readable? I'd expect any tooling requirement to override humans reading the output. ie, i think if you're going to push the output to a tool that tool should take care of the human readable bit too. whether that's a summary, or reformatting the raw results.

ie, i'd expect `$ benchmark_my_stuff --benchmark_format=json | analyse_my_bench`

to provide me with human readable analysis results. if i need the output for debugging, then `tee` or some equivalent would at least give me something to look at after the fact.

```
$ benchmark_my_stuff --benchmark_output=out.json --benchmark_output_format=json --benchmark_format=csv
```

it's very easy to get wrong. i mean, output is also to stdout, so output_format vs format is subtle.

this may just be bikeshedding the arguments, i guess. i'm wary of redundancy and confusion.